chapter8.adoc

```diff
@@ -1,37 +1,146 @@
 [[chapter8]]
 [[Smsdedbg]]
-== `Smsdedbg`: Supervisor Domain External Debug
+== Supervisor Domain External Trace and Debug
 
-The <<MSDCFG>> CSR contains the `sdedbgalw` bit that controls whether the
-current scheduled SD is allowed to be external-debugged cite:[ExtDbg]. This bit
-is context switched (along with rest of the `msdcfg`) per SD.
+The <<MSDCFG>> CSR contains the `sdedbgalw` and `sdetrcalw` bits that control
+if the current scheduled SD is allowed to be external-debugged cite:[ExtDbg]
+or to be external-traced, respectively. External trace refers to tracing
+software running on a separate target RISC-V platform, via a trace control/data
+transport that provides external access to the trace encoder controls
+cite:[ETrc]. This extension only affects debug or trace orchestrated by an
+external actor. Self-hosted debug (and trace) are managed by the RDSM.
 
-=== `Smsdedbg` interaction with external debug security controls (Informative)
+=== `Smsdedbg`: External Debug allowed control for Supervisor Domain
+
+When M-mode external debug is enabled, all supervisor domains may also be
+debugged by an external debugger irrespective of the configuration held in
+`msdcfg.SDEDBGALW`.
+
+When M-mode external debug is disabled, whether execution at privilege modes
+less than `M-mode` may be debugged by an external debugger depends on the
+configuration held in `msdcfg.SDEDBGALW`.
+
+When `msdcfg.SDEDBGALW` = 0, external debug is disallowed for the SD. Abstract
+commands and halt request from the debug module are suppressed and stay pending
+while the SD is active, on a per-hart basis. Other debug operations (current or
+defined in the future) must similarly be kept pending while the SD is active on
+a hart.
+
+When `msdcfg.SDEDBGALW` = 1 then external debug of privilege modes less than
+`M-mode` is allowed for the SD on a per-hart basis, and:
+
+* A valid request e.g. `haltreq` may transition the hart to Debug Mode. See
+other cases in RISC-V Debug specification cite:[ExtDbg].
```

Comment:
```suggestion
* A valid request (e.g. `halt`) or a trigger matching/firing may transition the hart to
Debug Mode. See other cases in RISC-V Debug specification cite:[ExtDbg]. 
Note that the `resethaltreq` is not applicable since it can be done only
following a reset and the hart cannot be at privilege mode less than
`M-mode` with `msdcfg.SDEDBGALW`=1 at de-assertion of reset.
```